soc/silabs/silabs_siwx91x/Kconfig

```diff
@@ -14,3 +14,21 @@ config SOC_SILABS_SLEEPTIMER
 	bool
 	help
 	  The Sleeptimer HAL module is used for SIWX91X.
+
+if SOC_FAMILY_SILABS_SIWX91X
+config SIWX91X_SILABS_COMMANDER_SIGN
+	bool "Silicon Labs Commander Signing Support"
+	depends on BUILD_OUTPUT_BIN
+	help
+	  Activates signing of Zephyr binary with Silicon Labs Commander during build process.
+
+config SIWX91X_OTA_KEY
+	string "OTA Key for Encryption (sourced from prj.conf)"
```

Comment:
You don't need to mention `prj.conf`. `prj.conf` is a specific case.

BTW, is it specific to the OTA?